src/install/aws/footer.md

```diff
@@ -0,0 +1,5 @@
+</br>
+<hr>
+
+<div class="right">Are you having trouble with AWS Install? <a href="/install/local/install.md">Try local!</a>
```

Comment:
I feel like we can add a feedback block as well.  

Have feedback? Drop us a message at support@mydecisive.ai or fill out our form at <insert_link>